pkg/discovery/dtofactory/node_entity_dto_builder.go

```diff
@@ -221,11 +224,82 @@ func (builder *nodeEntityDTOBuilder) getNodeCommoditiesSold(node *api.Node, clus
 		Capacity(clusterCommodityDefaultCapacity).
 		Create()
 	if err != nil {
-		return nil, err
+		return nil, isAvailableForPlacement, err
 	}
 	commoditiesSold = append(commoditiesSold, clusterComm)
 
-	return commoditiesSold, nil
+	return commoditiesSold, isAvailableForPlacement, nil
+}
+
+func (builder *nodeEntityDTOBuilder) getNodeStorageCommoditiesSold(nodeName string) ([]*proto.CommodityDTO, bool) {
```

Comment:
Please add a comment for the method to explain the behavior of isAvailableForPlacement

Reply:
I generally try to use self documenting names and sometimes an additional variable to explain the purpose, especially for evaluating conditionals like this. For example as done [here](https://github.com/turbonomic/kubeturbo/pull/537/files#diff-c8b9a819f9d9f29527ec258b2617a74fe62e1645dab6805fb4ab05991f2d1e3eR300) for this variable.
I have anyways added a comment at the top of the function also now.